go/vt/vtgate/executor_test.go

```diff
@@ -1257,6 +1280,66 @@ func TestExecutorCreateVindexDDL(t *testing.T) {
 	}
 }
 
+func TestExecutorAddDropVschemaTableDDL(t *testing.T) {
```

Comment:
nit ~ I think this is clearer `TestExecutorAddDropVschemaUnshardedTableDDL`.

Also, shouldn't we support this method to drop a table from a sharded vschema as well ?